dcos/http.py

```diff
@@ -43,8 +43,10 @@ def _verify_ssl(verify=None):
     :rtype: bool | str
     """
 
-    if verify is None and constants.DCOS_SSL_VERIFY_ENV in os.environ:
-        verify = os.environ[constants.DCOS_SSL_VERIFY_ENV]
+    ssl_verify_config = util.get_config().get("core.ssl_verify")
+    if verify is None and (constants.DCOS_SSL_VERIFY_ENV in os.environ or
+                           ssl_verify_config is not None):
+        verify = os.environ[constants.DCOS_SSL_VERIFY_ENV] or ssl_verify_config
```

Comment:
We should standardize the precedence order and have a mechanism that enforces this order. E.g. environment variable > config file. We have this pattern littered in a few places in the code.